forUncrustifySources.cfg

```diff
@@ -424,6 +425,7 @@ nl_func_def_args                          = ignore
 # nl_property_brace
 # nl_scope_brace
 # nl_synchronized_brace
+# nl_template_class
```

Comment:
Eh, okay, but I *think* this is superfluous given that `nl_template_class_decl` and `nl_template_class_def` are both specified. (I'd have to go back and check whether `nl_template_class` can ever be applied when both of those are not `ignore`.)

Reply:
The lines are comments.
I think this is an help to see that all options where tested, if used with a tested value,
if not used for the uncrustify sources, as a comment.